pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
   "fsspec",
   "requests",
   "treelib",
+  "pydantic",
```

Comment:
Let's confirm if this means that custom probes can use pydantic etc.? Could be valuable to document somewhere that probes can depend on anything that juju doctor already depends on.